examples/experiments/level_test.py

```diff
@@ -3,7 +3,7 @@
 Sound level test and visual size calibration
 ============================================
 
-This example tests the audio level and video size. For audio, it produces a 65
+This example tests the audio level and video size. For audio, it produces an 80

```

Comment:
More signal relative to noise floor (sound level meter and room noise). Seemed like the dB reading was more stable when testing in 226 yesterday.